coalib/testing/LocalBearTestHelper.py

```diff
@@ -117,11 +117,11 @@ def check_results(self,
                               list,
                               msg="The given results are not a list.")
 
-        with prepare_file(lines, filename, settings,
+        with prepare_file(lines, filename,
                           force_linebreaks=force_linebreaks,
                           create_tempfile=create_tempfile,
                           tempfile_kwargs=tempfile_kwargs) as (file, fname), \
-                execute_bear(local_bear, fname, file) as bear_output:
+                execute_bear(local_bear, fname, file, settings) as bear_output:
```

Comment:
ah sorry for not mentioning, I think you need to unpack settings:
```
execute_bear(..., **settings)
```
look at the `execute_bear` definition on how to use it :)